Header header
uint32 id
geometry_msgs/PoseStamped pose
geometry_msgs/Point[] corners
